src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/tests/stubs/stub_deployment_map.yml

```diff
@@ -1,230 +0,0 @@
-pipelines:
```

Comment:
The content of the file was an out of date copy of the [`${bootstrap-repo}/adf-bootstrap/deployment/lambda_codebase/initial_commit/pipelines_repository/example-deployment_map.yml`](https://github.com/awslabs/aws-deployment-framework/blob/991a076ec70972acebeaf45c6a40de9f8ec681a4/src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/lambda_codebase/initial_commit/pipelines_repository/example-deployment_map.yml)

So instead, I made this a symbolic link so they cannot go out of sync from now on.
See the next file in this review, with a similar name. Unfortunately GitHub doesn't display this diff clearly.